coverage/generate-index-html.py

```diff
@@ -32,18 +32,36 @@
         main { margin-bottom: 2rem; }
         .table-header,
         .table-row {
+          box-sizing: border-box;
           display: flex;
           width: 100%;
           padding: 2px 10px;
         }
-        .table-header { font-weight: bold; }
+        .table-header { font-weight: bold;}
```

Comment:
You'll need to make another PR. Honestly, I'm not so sure you need to fix that, although by all means, do so if you want to. (There's another line further down with the same issue, for what it's worth, so probably fix them both.)